lib/mixlib/install/version.rb

```diff
@@ -1,5 +1,5 @@
 module Mixlib
   class Install
-    VERSION = "0.8.0"
+    VERSION = "1.0.0"
```

Comment:
I would like the change the version in master to 0.8.0-pre or 0.8.0-dev and let it be.  We can change it to whatever version we want when we release.